changelog/3583.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix encoding error with `print` statements in doctests
```

Comment:
This should be `rst`:

```
Fix encoding error with ``print`` statements in doctests.
```

Reply:
It is RST; reStructuredText uses single backticks for individual identifiers, while double backticks are for arbitrary inline code.

Reply:
Oh you are right, sorry; 

We use inline code in the changelog though because there won't be any references to it in the final document (but TBH I don't remember if they render differently).